linux_os/guide/system/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_fchmod/rule.yml

```diff
@@ -58,7 +58,7 @@ references:
     stigid@ol7: OL07-00-030410
     stigid@ol8: OL08-00-030540
     stigid@rhel7: RHEL-07-030420
-    stigid@rhel8: RHEL-08-030540
```

Comment:
Should the comments in the profile be removed too?
https://github.com/ComplianceAsCode/content/blob/77730337dd834cefd8e7e8d410d60aa043bd42e2/products/rhel8/profiles/stig.profile#L857

Reply:
That's correct

Reply:
Done in d871cc8379ecf77429e651df104445c719543f03